app/assets/javascripts/patients.coffee

```diff
@@ -9,3 +9,20 @@ $(document).on "click", "#toggle-call-log", ->
 
 $(document).on "change", ".edit_patient", ->
   $(this).submit()
+
+$(document).on "click", ".add-button", ->
+  value = $(".pledges-select").val()
+  if value == ""
+    alert("You have to select a value from the dropdown menu!")
+  else
+    html = """
+          <div class="form-group">
```

Comment:
Do you think there's a way we can turn this into an `html.erb` partial we can append? I think I'd prefer that to something hidden in the js assets, but I'm not married to it -- what are your thoughts?